### DISCUSSION TITLE
how do I apply partial transparency to my colormap?

### DISCUSSION THREAD

**Discussion by hrodmn** [cogeotiff/rio-tiler#663] [Q&A] [answered]
I am experimenting with custom `colormaps` (via `titiler-pgstac`) and I am having trouble when I try to use the `alpha` value to control transparency.

Here is the colormap that I want. It should be 50% transparent brown and blue:
```python
{
    0: (216, 179, 101, 127),
    1: (90, 180, 172, 127)
}
```

When I set `alpha` to anything greater than 1, I get fully opaque colors like this:
![image](https://github.com/cogeotiff/rio-tiler/assets/17726304/51b0dd7c-c924-439b-84f5-a45d6f6b6823)

When I set `alpha` to anything less than 1, the color comes back 100% transparent like this:

![image](https://github.com/cogeotiff/rio-tiler/assets/17726304/df2a030e-f92a-47ce-a102-69667d2b0ddb)

`colormap` input:
```python
{
    0: (216, 179, 101, 0.1),
    1: (90, 180, 172, 127)
}
```

Is there something wrong with the alpha interpretation or is this the expected behavior?

**Comment by vincentsarago** [comment, marked as answer]
🤔 is this using titiler or rio-tiler? 

I thought we fixed this in titiler https://github.com/developmentseed/titiler/issues/689

**Reply by hrodmn** [reply]
Ah ha, thank you @vincentsarago! I guess I don't fully understand the dependency structure between titiler / rio-tiler. I am pinned to `titiler-pgstac==0.5.1` still, which I think is pinned to `titiler>=0.12,<0.13` so now I have one more reason to upgrade to a more modern version.
